src/components/Carousel/hooks/useCarousel.ts

```diff
@@ -36,9 +74,11 @@ function setAriaAttributes(
       ) {
         item.current?.setAttribute('aria-hidden', 'true');
         item.current?.setAttribute('tabIndex', '-1');
+        setButtonsFromCardTabIndex(item.current, true);
```

Comment:
Based on React conventions, I believe this should be implemented as state on the individual cards, updating the tabIndex in the re-render.

Reply:
Done